CONTRIBUTING.md

```diff
@@ -0,0 +1,253 @@
+# Contributing to lisk-template
+
+First off, thanks for taking the time to contribute! :raised_hands:
+
+The following is a set of guidelines for contributing to lisk-template, which
```

Comment:
This file needs to be customised for each individual project. At the very least `lisk-template` needs to be replaced with the name of your project, but ideally you would also add project-specific examples/extensions. Take a look at Commander's `CONTRIBUTING.md` for an example.